config/locales/en.yml

```diff
@@ -1328,7 +1329,7 @@ en:
     p_id: "Protocol ID:"
     p_info: "Protocol Information"
     proxy: "SPARC Proxy Rights"
-    received: "You have received a new notification on your SPARCRequest User Dashboard. Please login  @ %{portal_root_url} to retrieve it. Thank you."
+    received: "You have received a new notification on your <a href='%{portal_root_url}/protocols/%{protocol_id}'>SPARCDashboard</a> in regard to protocol (%{protocol_id})"
```

Comment:
It would probably be better to pass the `href` link directly and set `link = dashboard_protocol_path(@protocol)` where it's assigned

Reply:
thanks Kyle, done!